tests/wasm/runtime/index.ts

```diff
@@ -109,9 +114,8 @@ function imports(abi: () => Abi, host: Host): WebAssembly.Imports {
       'log.log': (level: number, message: Pointer) => {
         host.log.log(level, readStr(message))
       },
-      'store.get': () => {
-        // TODO(nlordell): Implement storage reading
-        return 0
+      'store.get': (entity: Pointer, id: Pointer) => {
+        return writeEntityOrNull(host.store.get(readStr(entity), readStr(id)))
```

Comment:
Why do we need to write the entity upon reading?

Reply:
We need to write the result to the subgraph Wasm module's memory, since it is the one doing the reading.